### ISSUE TITLE
Add support for extra NaiveDate Chrono type

### ISSUE THREAD

**Issue by JasterV** [juhaku/utoipa#281]
I'm trying to use this library along with the `chrono` and I use the `NaiveDate` on lots of places but it is not supported by `utoipa`
 
> Could not resolve reference: Could not resolve pointer: /components/schemas/NaiveDate does not exist in document

**Comment by JasterV** [comment]
@juhaku Thanks for your fast reply :D We were able to solve the errors on the schemas by using the `value_type` as you are saying.

But now we have another problem, we use `NaiveDate` as a param attribute on an endpoint and we are getting the same error as before.

**Comment by juhaku** [comment]
Anytime :slightly_smiling_face: If you are using `IntoParams` then you are able to use the same `value_type` attribute for param as well. See the docs here: https://docs.rs/utoipa/latest/utoipa/derive.IntoParams.html

**Comment by JasterV** [comment]
Cool! @juhaku

This is my current solution:

```
#[derive(Deserialize, IntoParams, Debug)]
pub struct Params {
    #[param(allow_reserved, value_type = String, example = "2022-01-22")]
    pub claim_date: NaiveDate,
    #[param(example = "Victor")]
    pub name: String,
}
```

I would love to be able to add a description for each field but I can't find the way to do it. Is there a way I can support both having the NaiveDate & having a description? 
Also being able to specify the format of the String such as the Date would be awesome :smile:

**Comment by juhaku** [comment]
Description is actually just a doc comment above the field. While it might not be most explicit but it should be described somewhere at the docs, but might be missing. Seems by a quick look the details regarding the description is missing from the docs. Description is only supported by primitive fields or inlined objects. `$ref` fields to other schemas does not have description support. 
```rust
#[derive(Deserialize, IntoParams, Debug)]
pub struct Params {
    /// This first line here at the doc comment will end up being description in the OpenAPI doc
    /// 
    /// More additional details to comment
    #[param(allow_reserved, value_type = String, example = "2022-01-22")]
    pub claim_date: NaiveDate,
    #[param(example = "Victor")]
    pub name: String,
}
```

**Comment by JasterV** [comment]
@juhaku Perfect thank you so much! Then I think we can close this issue :smile: 

Pd: As said, could it be possible to specify the format of the `value_type`? Like `format = Date`. 
We've done it when defining components but it can't be done here

**Comment by juhaku** [comment]
Ah, that is missing feature. There should be done some unification within the library to unify the functionality between `IntoParams` and `ToSchema` I'll add a new issue regarding this. Good finding

**Comment by juhaku** [comment]
> @juhaku Perfect thank you so much! Then I think we can close this issue :smile: 

Yeah can  be closed I suppose.

**Comment by JasterV** [comment]
@juhaku Thank you man! Love how fast at replying and the effort you are putting here :smile:

**Comment by juhaku** [comment]
@JasterV chrono `NaiveDate` will be natively supported from next release. There is merged PR #404 for it.
